apps/wallet-mobile/src/features/RampOnOff/common/AmountCard/AmountCard.tsx

```diff
@@ -193,6 +190,9 @@ const getStyles = (props: {theme: Theme}) => {
       color: theme.color.magenta[500],
       fontSize: 12,
     },
+    placeholderColor: theme.color.gray[600],
+    textFocused: theme.color.gray[900],
+    textBlurr: theme.color['black-static'],
   })
   return styles
```

Comment:
@SorinC6 we can't touch the `styles` created by the `StyleSheet`, if we add non-style properties, later when "babel" into a browser style it is doom to fail, so non-style properties need to be returned a part from the styles, follow my suggestion.
```suggestion
  })
  const colors = {
    placeholder: theme.color.gray[600],
    focused: theme.color.gray[900],
    blur: theme.color['black-static'],
  }
  return {styles, colors} as const
```

Reply:
an alternative would be to use the text style `placeholderText: { color: ...}` but the lint for unused styles is dumb.